content/docs/serverless-workflows/troubleshooting.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Troubleshooting
+date: "2024-07-25"
+weight: 101
+---
+
+
+# Troubleshooting Guide
+
+This document provides solutions to common problems encountered with serverless workflows.
+
+## Table of Contents
+
+1. [Workflow Errors](#workflow-errors)
+2. [Configuration Problems](#configuration-problems)
+3. [Performance Issues](#performance-issues)
+4. [Error Messages](#error-messages)
+5. [Network Problems](#network-problems)
+6. [Common Scenarios](#common-scenarios)
+7. [Contact Support](#contact-support)
+
+---
+
+## Workflow Errors
+
+### Problem: Workflow execution fails
+
+**Solution:**
+
+1. Examine the container log of the workflow
+    ```console
+        kubectl logs my-workflow-xy73lj
+    ```
+
+### Problem: Workflow is not listed by the orchestrator plugin
+
+**Solution:**
+
+1. Examine the container log of the workflow
```

Comment:
First, we should check for the pod status which might not be started.
Perhaps the first step should be checking the sonataflow status?
Then checking the status of the pod what might not be started and therefore no logs.

Reply:
This is when the workflow executed but failed. 
There should be another entry for when the workflow immediatly fails in the UI